flagset.go

```diff
@@ -65,6 +65,12 @@ func (f *FlagSetFiller) walkFields(flagSet *flag.FlagSet, prefix string,
 		field := structType.Field(i)
 		fieldValue := structVal.Field(i)
 
+		if flagTag, ok := field.Tag.Lookup("flag"); ok {
```

Comment:
Overloading `flag` with an empty value doesn't seem quite intuitive to me. At the same time I'm hesitant to introduce another tag since they aren't namespaced...however, I'm leaning towards the latter like `ignore-flag` or `skip-flag`.

Reply:
I used the `flag` tag because currently it doesn't operate on a field that has children. If you want to ignore a nested chain you have to put an empty `flag` tag on every child. This just shortcuts that loop by ignoring the nested chain from the start. 

If you still want to add a new flag, I can do that.

Reply:
Sorry, I went back and read the documentation and now see `flag:""`  was already a feature. I'm good to go as it is.